### PR TITLE
Added npm support

### PR DESCRIPTION
**Pull request by rodolfodosanjos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ajax",
```
Comment (renie) [inline package.json]: The name must be the same on bower.json.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "version": "1.0.0",
```
Comment (renie) [inline package.json]: This version should be the same version that is on bower.json. And as this is a new feature, but does not break anything, the third number of version must be increased. So 1.0.4 for both.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "description": "A tiny XMLHTTPRequest abstraction",
+  "main": "Ajax.js",
```
Comment (renie) [inline package.json]: main file shoud be the minified version

File: package.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  "scripts": {
```
Comment (renie) [inline package.json]: we have no test yet. so there is no need for scripts section now

File: package.json (hunk 1 above)
Comment (rodolfodosanjos) [inline package.json]: NPM just accepts lowercase names

File: package.json (hunk 3 above)
Comment (rodolfodosanjos) [inline package.json]: bower.json is not using using the minified version, should I change both to the minified one?

File: package.json (hunk 1 above)
Comment (renie) [inline package.json]: ok then

File: package.json (hunk 3 above)
Comment (renie) [inline package.json]: yes please

**Pull request merged**